crap-advisor-frontend/src/core/components/Sidebar/index.jsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import "./index.css"
-import logo from "../../../assets/logo.png"
+import Review from "./Review";
+import {apiUrl} from "../../../app/constants";
+
+const Sidebar = ({selectedRestroom}) => {
+    const [reviews, setReviews] = useState([])
+
+    useEffect(() => {
+        if (selectedRestroom != null) {
+            fetch(apiUrl + "/review/restroom/" + selectedRestroom.id)
+                .then((response) => response.json())
+                .then((data) => {
+                    setReviews(data)
+                    console.log(`Received ${data.length} reviews for restroom with id ${selectedRestroom.id}`)
```

Comment:
Please, remove logs from code. It's only for debug

Reply:
Yeah i got it